### PR TITLE
Added a page for the ghost student

### PR DESCRIPTION
**Pull request by nw0428**
Hey guys please review this

### CODE REVIEW

**Review by chanceencounter — APPROVED** [review]
Reasonable.

File: html/ghost.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
```
Comment (mklait) [inline html/ghost.html:3]: looks good to me. go for it.

**Review by ericywong — APPROVED** [review]
looks good.

**Comment by MSevey** [comment]
i'm afraid of ghosts

**Review by mklait — APPROVED** [review]
i guess this works.

**Comment by bwrobid** [comment]
looks good

**Pull request merged**